Tiltfile

```diff
@@ -127,11 +128,23 @@ helm_resource(
     poktroll_chart,
     flags=[
         "--values=./localnet/kubernetes/values-common.yaml",
+        "--values=./localnet/kubernetes/values-relayer.yaml",
```

Comment:
<img width="864" alt="Screenshot 2023-11-14 at 19 44 45" src="https://github.com/pokt-network/poktroll/assets/53987565/c4c9f7f7-cb7c-46e9-b7fc-acd0f1e0ef1f">

Still getting this error with the relayers in tilt. Is it still looking for the old command and not the new one?

Reply:
I am surprised you still see that, as we no longer have the `pocketd` binary.

Reply:
Confirmed: this would not happen on a PR with `relayminer` implemented. We just need to merge everything.